.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,14 +4,14 @@ about: Create a report to help us improve
 title: ''
 labels: 'bug'
 assignees: ''
-
 ---
 
 **Describe the bug**
 A clear and concise description of what the bug is.
 
 **To Reproduce**
 Steps to reproduce the behavior:
+
```

Comment:
I understand this was the work of the `linter`, but I don't think there should be a `newline` here.